packages/commonwealth/client/scripts/views/SublayoutHeader.tsx

```diff
@@ -46,7 +46,7 @@ export const SublayoutHeader = ({ onMobile }: SublayoutHeaderProps) => {
      * Imp to reset wc session on logout as subsequent login attempts fail
      */
     const walletConnectWallet = WebWalletController.Instance.getByName(
-      WalletId.WalletConnect
+      WalletId.WalletConnect,
```

Comment:
what kind of the config do you use for prettier? Because this and other lines in this PR where you add `,` at the end of the line are not correct (please revert). Please make sure you run `yarn setup-git-hooks` and  `yarn add-prettier-precommit` on your local machine